metallb/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: config_metallb start_frr
+
+all:
+	echo "make {config_bgp | config_l2}"
+
+config_bgp: config_metallb_bgp start_frr
```

Comment:
Maybe `start_frr` should come before `config_metallb_bgp`?  To have the cleanest startup, we'd want frr running before deploying metallb.

You could make `start_frr` a dependency on `config_metallb_bgp`

```
config_bgp: start_frr
    ./metallb_bgp.sh

start_frr:
    ./start_frr.sh
```

Reply:
Changed.